examples/layeredeffects/DijkstraStateMonad.fst

```diff
@@ -149,7 +149,8 @@ let lem_on_comp #a #b #c (f:b -> c) (g:a -> b)
   = ()
 
 /// We also rely on eta being a provable equality
-let eta (f:'a -> 'b) : Lemma (f == (fun x -> f x)) = ()
+/// AR: 05/11, eta is no longer provable, so admitting it
+let eta (f:'a -> 'b) : Lemma (f == (fun x -> f x)) = admit ()
```

Comment:
This file is just a sketch with many assumes in it already. 

It's not worth fixing the proofs here to not use eta.